spec/requests/bundles/edit_spec.rb

```diff
@@ -224,7 +224,9 @@
           click_on "Remove"
         end
         within_cart_item "Versioned product" do
-          expect(page).to have_text("Qty: 1")
+          within("[aria-label='Quantity']") do
```

Comment:
I forget if Capybara picks up `sr-only` text, but that might be a better solution for quantity in general. We try to avoid specific selectors like this